examples/light-switch-app/nrfconnect/README.md

```diff
@@ -577,7 +583,35 @@ To perform binding process you need to complete following steps:
     __{"fabricIndex": 1, "node": <1>, "endpoint": 1, "cluster": 8}__ is a binding for __LevelControl__ cluster. 
 
 
-To test communication between Light Switch and bound device use [buttons](#buttons) or [CLI commands](#matter-cli-commands). Both of them are described in [Device UI section](#device-ui).
+### Binding multicast devices using chip-tool for Windows/Linux
+Binding multicast allows to add all lighting devices to one multicast group. After that Light Switch can send multicast request and all of devices listening bound multicast group can run received command. It can be used to control more than one lighting device via single light switch at the same time.
+
+ITo perform binding for group multicast you need to complete following steps:
+
+1. Navigate to the CHIP root directory:
+
+2. Build chip-tool using [chip-tool guide](../../../docs/guides/chip_tool_guide.md#building).
+
+3. Go to chip-tool build directory.
+
+4. Add ACL (Access Control List) in the lighting endpoint permissions to receive commands from Light Switch:
```

Comment:
Overall comment to the all point below. Please use some variables to describe endpoint ids and node ids in the commands. I know that by default we have the binding cluster on the endpoint 1, but user may modify it in the code and it would be nice to have generic commands for that.

Reply:
Yes, but we decided to inform a user that we commissioned a lighting app with node id 1 and light-switch with node id 2 and then use these values in the following codes. However, I am still confused about which form is suitable for this example. So, @greg-fer @Damian-Nordic, what do you think?

Reply:
I think it's easier for a user to follow a guide if he can see specific values in the command-line, and I'm not sure if we be prepared for any modification a user does to the sample. Maybe we could just describe before the command what subsequent arguments mean?